src/GUI/CafeBill.java

```diff
@@ -769,6 +783,54 @@ public void actionPerformed(ActionEvent e) {
 				// Change Values customerId, transID, transInfo from the Credit Cash Dialog
 				setMenuOrder(CafeBill.cid, Payment.payCashOrCC, Payment.transInfo, Float.parseFloat(lblDiscount.getText()), (float)CouponDiscount.couponValue, (String)CouponDiscount.DISCOUNTDEC, Float.parseFloat(lblSubtotal.getText()), db_totalTaxPerc, Float.parseFloat(lblTotal.getText()));
 				int iRowCnt = dataModel.getRowCount();
+				
+				Connection connect = null;
+				Connection con = ConnectionManager.getConnection();
+			    PreparedStatement preparedStatement = null;
+			    String sql=null;
+
+				int ToV = 0;
+				int NoV = 0;
+
+				try{
+					// first get the total no. of visits in current table;
+					sql = "SELECT TotalNoVisits FROM customer WHERE cid = "+cid;
+					System.out.println(sql);
+					preparedStatement = connect.prepareStatement(sql);
+					System.out.println(sql);
+					resultSet = preparedStatement.executeQuery();
+					System.out.println("after result set 64");
+					System.out.println("before visits ="+resultSet.getFetchSize());
+
+					while(resultSet.next()){
+						ToV = resultSet.getInt(1);
+					}
+
+					//select no. of visits from customer current reward table
+					sql = "SELECT NumberOfVisits from CustomerCurrentReward where custId = "+cid;
+					preparedStatement = connect.prepareStatement(sql);
+					System.out.println(sql);
+					resultSet = preparedStatement.executeQuery();
+					while(resultSet.next()){
+						NoV=resultSet.getInt(1);
+					}
```

Comment:
Also comment or delete this block of code it refers to CustomerCurrentReward that is not present in his commit.